homeassistant/components/knx/light.py

```diff
@@ -275,13 +294,14 @@ def supported_features(self):
                 update_color_temp:
             # calculate relative_ct from Kelvin to fit typical KNX devices
             kelvin = int(color_util.color_temperature_mired_to_kelvin(mireds))
-            relative_ct = int(255 * (kelvin - self._min_kelvin) /
-                              (self._max_kelvin - self._min_kelvin))
+            relative_ct = int(255 * (kelvin - self._min_kelvin) / (
+                self._max_kelvin - self._min_kelvin))
             await self.device.set_tunable_white(relative_ct)
         else:
             # no color/brightness change requested, so just turn it on
             await self.device.set_on()
 
     async def async_turn_off(self, **kwargs):
         """Turn the light off."""
+        del kwargs
```

Comment:
Why is this done?

Reply:
@phbaer could you have a look, please?

Reply:
I have tested it with this line removed, as I have no   idea what it is supposed to do. Everything continues working, but I have no   RGBW lights to test against - this was added with the RGBW feature - https://github.com/XKNX/xknx/pull/206

Reply:
Just remove it. :+1: